### PR TITLE
[REVIEW] Fix for empty folders

### LINKED ISSUE #1070: [BUG] The engine crashes when we try to create a table from an empty directory
**Describe the bug**
The engine crashes when we try to create a table from an empty directory.

**Steps/Code to reproduce bug**
The reproducer is as follow:
```
from blazingsql import BlazingContext
bc = BlazingContext()
bc.create_table('lineitem', '/home/ubuntu/tpch/') # this is an empty dir
```

**Expected behavior**
An error should be raised with no crash.

**Environment overview (please complete the following information)**
 - Environment location: Bare-metal
 - Method of cuDF install: From source branch-0.16

### PR DESCRIPTION
**Pull request by rommelDB**
Fixes #1070

### CODE REVIEW

**Comment by rommelDB** [comment]
rerun tests

File: engine/src/io/data_provider/UriDataProvider.cpp [hunk 1]
```diff
@@ -150,18 +150,45 @@ data_handle uri_data_provider::get_next(bool open_file) {
 				this->directory_uris = BlazingContext::getInstance()->getFileSystemManager()->list(target_uri);
 			}
 
-			std::string ender = ".crc";
+			// sometimes parquet directories have somes files that
+			// have not the same schema as the *.parquet files
+			// we don't want the data provider handle this ones
+			std::vector<std::string> ignored_suffixes{".crc", "_metadata", "_SUCCESS"};
```
Comment (felipeblazing) [inline engine/src/io/data_provider/UriDataProvider.cpp:156]: we should add ipython notebook checkpoints to this those get made everywhere

**Review by felipeblazing — APPROVED** [review]

File: engine/src/io/data_provider/UriDataProvider.cpp (hunk 1 above)
Comment (felipeblazing) [reply engine/src/io/data_provider/UriDataProvider.cpp:156]: .ipynb_checkpoints
are the files itmakmes

Comment (romeluko) [reply engine/src/io/data_provider/UriDataProvider.cpp]: Added. Thank you.

Comment (rommelDB) [reply engine/src/io/data_provider/UriDataProvider.cpp]: Added, thank you.

**Pull request merged**